fh-android-sdk/pom.xml

```diff
@@ -26,6 +26,7 @@
   <properties>
     <loopj.version>1.4.8</loopj.version>
     <aerogear.android.push.version>2.2.1</aerogear.android.push.version>
+    <aerogear.android.store.version>3.0.0-SNAPSHOT</aerogear.android.store.version>
```

Comment:
Okay, but this is a PR against MASTER rather than a development branch. Unless you define Master as the development head and have a separate "stable" branch but I don't see that in this repository.